MicrosoftTeams.schema.json

```diff
@@ -1116,9 +1116,9 @@
           "type": "array",
           "uniqueItems": true
         },
-        "videoAppContentUrl": {
+        "videoFiltersConfigurationUrl": {
           "type": "string",
-          "description": "The url which indicates A/V filters' video app content.",
+          "description": "The url which indicates video filters' video app configuration.",
```

Comment:
A URL for configuring the video filters.

Reply:
Updated